checkstyle-tester/diff.groovy

```diff
@@ -74,8 +74,10 @@ def getCliOptions(args) {
             + ' as a shorter version to prevent long paths. (optional, default is false)')
         m(longOpt: 'mode', args: 1, required: false, argName: 'mode', 'The mode of the tool:' \
             + ' \'diff\' or \'single\'. (optional, default is \'diff\')')
-        xm(longOpt: 'extraMvnRegressionOptions', args: 1, required: false, 'Extra arguments to pass to Maven' \
-            + 'for Checkstyle Regression run (optional, ex: -Dmaven.prop=true)')
+        ce(longOpt: 'continueOnError', required: false, 'Whether to fail or continue the Checkstyle' \
+            + ' run when it reports a non-zero return code. (optional, default is false)')
+        xo(longOpt: 'extraRegressionOptions', args: 1, required: false, 'Extra arguments to pass ' \
+            + 'for Checkstyle Regression run (optional, ex: -Dprop=true)')
```

Comment:
Please do `Checkstyle Regression run` -> `Checkstyle execution`, it is better to keep it simple. We would still use these options if we weren't "regression" testing, too.

Reply:
Done.